manual_flagging/requirements.txt

```diff
@@ -1,4 +1,4 @@
-awswrangler==2.20.1
+awswrangler==3.7.3
```

Comment:
Bumped this because I was getting some sort of parquet error. It fixed the issue.